### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by romaroid**

### CODE REVIEW

**Comment by afk11** [comment]
Needs rebase

**Comment by rubensayshi** [comment]
testing

File: gulpfile.js [hunk 1]
```diff
@@ -376,6 +397,13 @@ gulp.task('watch', function() {
     gulp.watch(['./appconfig.json', './appconfig.default.json'], ['default:livereload']);
 });
 
+gulp.task('test', function(done) {
```
Comment (rubensayshi) [inline gulpfile.js]: `test` should have dependancy for `['js', 'templates']` I think?

File: src/js/modules/core/services/wallet/wallet.service.spec.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                }));
+
+                // Inject the service
+                beforeEach(inject(function($injector) {
+                    // service = $injector.get("walletsManagerService");
+                }));
+
+                /*it("Should be TRUE", function () {
+                    expect(true).toBe(true);
+                });*/
```
Comment (rubensayshi) [inline src/js/modules/core/services/wallet/wallet.service.spec.js]: remove dead code

**Pull request merged**